docs/docs/tokenizer/thai_tokenizer.md

```diff
@@ -0,0 +1,16 @@
+The `ThaiTokenizer` is a Rasa compatible tokenizer for Thai, using [`PyThaiNLP`](https://github.com/PyThaiNLP/pythainlp) under the hood.
+
+Once `pythainlp` is installed (e.g. via `pip install pythainlp`), it can be used in a Rasa configuration like below:
```

Comment:
Since `pythainlp` is added as a dependency, is this still needed?

Reply:
It might be nice to mention here that your pipeline example would/should break if the `language` setting is not set to `th`.

Reply:
right, I've just pushed the doc changes :)